server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -93,6 +95,7 @@
 
 @TestLogging(reason = "these tests do a lot of log-worthy things but we usually don't care", value = "org.elasticsearch:FATAL")
 public class CoordinatorTests extends AbstractCoordinatorTestCase {
+    private final Logger logger = LogManager.getLogger(CoordinatorTests.class);
```

Comment:
nit: unused? Also `org.elasticsearch.test.ESTestCase#logger` is doing the same thing I think?